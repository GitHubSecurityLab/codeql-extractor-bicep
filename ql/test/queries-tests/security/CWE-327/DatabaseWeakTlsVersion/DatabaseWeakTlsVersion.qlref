security/CWE-327/DatabaseWeakTlsVersion.ql
