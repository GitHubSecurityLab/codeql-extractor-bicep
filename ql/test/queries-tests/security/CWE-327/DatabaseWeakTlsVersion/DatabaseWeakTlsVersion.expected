| app.bicep:8:24:8:28 | String | Database 'vulnerable-postgresql-tls10' is configured with weak TLS version '1.0'. Use TLS 1.2 or higher. |
| app.bicep:20:24:20:31 | String | Database 'vulnerable-mysql-tls11' is configured with weak TLS version 'TLS1_1'. Use TLS 1.2 or higher. |
| app.bicep:32:24:32:28 | String | Database 'vulnerable-sql-tls10' is configured with weak TLS version '1.0'. Use TLS 1.2 or higher. |
| app.bicep:43:24:43:32 | String | Database 'vulnerable-mariadb-tls11' is configured with weak TLS version 'TLS_1.1'. Use TLS 1.2 or higher. |
